pipelines/release/official_release.groovy

```diff
@@ -265,7 +265,7 @@ notify.wrap {
           job: 'scipipe/ap_verify',
           parameters: [
             string(name: 'DOCKER_IMAGE', value: stackResults.image),
-            string(name: 'DATASET_REF', value: gitTag),
```

Comment:
Does `verify_drp_metrics` above needs this as well?

Reply:
Good call.  Did that as well.